Issue 4153 - Added a CI test

aadhikar opened the pull request:
Bug Description: The numSubordinates value doesn't always match the number of direct subordinate(s)

Relates: https://github.com/389ds/389-ds-base/issues/4153

Reviewed by: ??

aadhikar committed:
Issue 4153 - Added a CI test

Bug Description: The numSubordinates value doesn't always match the number of direct subordinate(s)

Relates: https://github.com/389ds/389-ds-base/issues/4153

Reviewed by: ??

droideck commented on dirsrvtests/tests/suites/monitor/monitor_test.py:
```diff
@@ -145,6 +146,35 @@ def test_monitor_backend(topo):
+        4. Should be same
+    """
+
+    raw_objects = DSLdapObjects(topo.standalone, basedn='cn=monitor')
```
I think we have 'Monitor()' class for that case (lib389.monitor)...

aadhikar replied:
I can extract the numSubordinates value while using Monitor class but  I am not sure I can use filters with it.

droideck replied:
You should be able, I think

droideck reviewed (approved):
Besides the small issue - LGTM!

droideck reviewed (commented):
Okay, discard my previous comment. It's impossible right now.
I've opened an issue for improvement though - #4559

aadhikar committed:
Issue 4153 - Added a CI test

Bug Description: The numSubordinates value doesn't always match the number of direct subordinate(s)

Relates: https://github.com/389ds/389-ds-base/issues/4153

Reviewed by: droideck
